airflow/contrib/operators/gcs_download_operator.py

```diff
@@ -1,8 +1,9 @@
 import logging
+import os
 
 from airflow.contrib.hooks.gcs_hook import GoogleCloudStorageHook
 from airflow.models import BaseOperator
-from airflow.utils.decorators import apply_defaults
+from airflow.utils import apply_defaults
```

Comment:
ah man, i didn't notice this changed. i think when i synced my repository to master the .decorators went away. will go ahead and put it back.